src/test/java/org/jitsi/meet/test/MalleusJitsificus.java

```diff
@@ -101,29 +119,31 @@
                 // XXX I don't remember if/why these are needed.
                 .appendConfig("config.p2p.useStunTurn=true")
                 .appendConfig("config.disable1On1Mode=false")
+                .appendConfig("config.testing.noAutoPlayVideo=true")
 
                 .appendConfig("config.p2p.enabled=" + (enableP2p ? "true" : "false"));
-            ret[i] = new Object[] { url, numParticipants, timeoutMs, numSenders};
+            ret[i] = new Object[] { url, numParticipants, timeoutMs, numSenders, numAudioSenders, regions};
         }
 
         return ret;
     }
 
     @Test(dataProvider = "dp")
     public void testMain(
-        JitsiMeetUrl url, int numberOfParticipants, long waitTime, int numSenders)
+        JitsiMeetUrl url, int numberOfParticipants, long waitTime, int numSenders, int numAudioSenders, String[] regions)
         throws InterruptedException
     {
         Thread[] runThreads = new Thread[numberOfParticipants];
 
-        for(int i = 0; i < numberOfParticipants; i++)
+        for (int i = 0; i < numberOfParticipants; i++)
         {
             runThreads[i]
                 = runAsync(
-                    i,
                     url,
                     waitTime,
-                    i >= numSenders /* no video */);
+                    i >= numSenders /* no video */,
+                    i >= numAudioSenders /* no audio */,
+                    regions == null ? null : regions[i % regions.length]);
```

Comment:
@bgrozev do we want to add support for the "stacked sender" allocation as well as the evenly distributed one here?

Reply:
If we do we should probably split it out into a separate PR - in fact I'm wondering if @bgrozev's existing enhancements from this PR should be split out as well.

Reply:
I just modified the code in place for the "stacked sender" test, I don't have anything ready for committing.